lib/urn.rb

```diff
@@ -1,26 +1,88 @@
+def URN(urn)
+  URN.new(urn)
```

Comment:
Yeah, I was thinking about it. We don't support it but maybe is worth it. Will do.